coralproject/cay
# Form builder refactor

## What does this PR do?
- Cleans lots of code in the FormBuilder, aiming for a leaner implementation
## How do I test this PR?
- Go to **Create form**
- Play with the form builder, drag and drop fields, etc
- Other things to consider:
  - On drop, the field should expand automatically
  - Esc & Enter should do Cancel & Save anywhere on the field editor
  - Everything should work smoothly

@coralproject/frontend
